src/components/Trashy/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+import TrashySchedule from 'components/Trashy/TrashySchedule';
+import TrashyAddress from 'components/Trashy/TrashyAddress';
+
+const nextBulkPickupDate = new Date().toString();
+
+class Trashy extends Component {
+  state = {
+    address: null,
+    addressSuggestions: [],
+    pickupDates: [],
+    nextBulkPickupDate: null,
+  };
+
+  getAddressSuggestions = addressText => {
+    axios
+      .post('https://recollect-wrapper.herokuapp.com/api/suggest', {
+        address: addressText,
+      })
+      .then(response => this.setState({ addressSuggestions: response.data }))
+      .catch(error => console.log(error));
+  };
+
+  getCalendar = addressId => {
+    axios
+      .post('https://recollect-wrapper.herokuapp.com/api/calendar', {
+        id: addressId,
+      })
+      .then(response => this.setState({ pickupDates: response.data }))
+      .catch(error => console.log(error));
+  };
+
+  getNextBulkPickupDate = addressId => {
+    axios
+      .post('https://recollect-wrapper.herokuapp.com/api/nextbulkpickup', {
+        id: addressId,
+      })
+      .then(response => this.setState({ nextBulkPickupDate: response.data[0] }))
+      .catch(error => console.log(error));
+  };
+
+  getAddressId = address => {
+    axios
+      .post('https://recollect-wrapper.herokuapp.com/api/address', address)
+      .then(response => response.data)
+      .then(addressId => {
+        this.getCalendar(addressId);
+        this.getNextBulkPickupDate(addressId);
+      })
+      .catch(error => console.log(error));
+  };
+
+  setAddress = address => {
+    this.setState({ address: address });
+    this.getAddressId(address);
+  };
+
+  render() {
+    const { intl, options } = this.props;
+    return (
+      <div className="coa-Trashy">
+        <TrashyAddress
+          suggestions={this.state.addressSuggestions}
```

Comment:
Just curious why TrashyAddress doesn't manage fetching address suggestions and the returned list of addressSuggestions as well? It doesn't look like any other component needs to be aware of this data or fetching process but it's totally possible that I'm overlooking something